lib/ueberauth/strategy/cognito.ex

```diff
@@ -96,15 +91,12 @@ defmodule Ueberauth.Strategy.Cognito do
   end
 
   defp request_jwks(http_client) do
-    %{
-      aws_region: aws_region,
-      user_pool_id: user_pool_id
-    } = get_config()
+    config = Config.get_config()
```

Comment:
You already have a config from line 64: can you pass it in as an argument?

Reply:
And `http_client` (possibly even `jwt_verifier`) could also be part of that struct.